test/planner/plan_util_test.cpp

```diff
@@ -0,0 +1,170 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// plan_util_test.cpp
+//
+// Identification: test/planner/plan_util_test.cpp
+//
+// Copyright (c) 2015-18, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "common/harness.h"
+
+#include "catalog/catalog.h"
+#include "catalog/database_catalog.h"
+#include "catalog/index_catalog.h"
+#include "catalog/table_catalog.h"
+#include "common/statement.h"
+#include "concurrency/transaction_manager_factory.h"
+#include "executor/testing_executor_util.h"
+#include "parser/postgresparser.h"
+#include "storage/data_table.h"
+
+#include "planner/plan_util.h"
+
+namespace peloton {
+namespace test {
+
+#define TEST_DB_NAME "test_db"
+
+class PlanUtilTests : public PelotonTest {};
+
+TEST_F(PlanUtilTests, GetAffectedIndexesTest) {
+  auto catalog = catalog::Catalog::GetInstance();
+  catalog->Bootstrap();
+
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+
+  catalog->CreateDatabase(TEST_DB_NAME, txn);
+  auto db = catalog->GetDatabaseWithName(TEST_DB_NAME, txn);
+  // Insert a table first
+  auto id_column = catalog::Column(
+      type::TypeId::INTEGER, type::Type::GetTypeSize(type::TypeId::INTEGER),
+      "id", true);
+  auto fname_column =
+      catalog::Column(type::TypeId::VARCHAR, 32, "first_name", false);
+  auto lname_column =
+      catalog::Column(type::TypeId::VARCHAR, 32, "last_name", false);
+
+  std::unique_ptr<catalog::Schema> table_schema(
+      new catalog::Schema({id_column, fname_column, lname_column}));
+  txn_manager.CommitTransaction(txn);
+
+  txn = txn_manager.BeginTransaction();
+  catalog->CreateTable(TEST_DB_NAME, "test_table", std::move(table_schema),
+                       txn);
+  txn_manager.CommitTransaction(txn);
+
+  txn = txn_manager.BeginTransaction();
+  auto source_table = db->GetTableWithName("test_table");
+  oid_t col_id = source_table->GetSchema()->GetColumnID(id_column.column_name);
+  LOG_INFO("id: %d", static_cast<int>(col_id));
```

Comment:
Just curious, are these logs needed in the test? What information do they convey?

Reply:
Ah, these are the debug logs leftover from my testing. Not needed.